star-ui/src/index.css

```
@@ -1,11 +1,107 @@
-@import "tailwindcss";
+/* LAYOUT ENGINE: Tiling WM (react-grid-layout) */
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap');
```

Stylelint (17.3.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap'" (import-notation)

(import-notation)